new/LIT7098QeneAlam.xml

```diff
@@ -0,0 +1,97 @@
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p><foreign xml:lang="gez">Qǝne</foreign> without type indication in the manuscript. It consists of six rhymed lines
+                    (although these do not correspond to the lines in the manuscript layout), 
+                    meaning
+                    that it might be classified as a <foreign xml:lang="gez">śəllāse qəne</foreign> according to
+                    <bibl><ptr target="bm:Guidi1900Qene"/></bibl>. 
+                    It is attributed to <persName ref="PRS14579WaldaGabreel"/> in the manuscript.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+                <language ident="sv">Swedish</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="JK" when="2024-10-11">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7098QeneAlam" passive="UppQ1524o"/>
+                    <relation name="saws:isAttributedToAuthor" active="LIT7098QeneAlam" passive="PRS14579WaldaGabreel"/>
+                </listRelation>
+            </div>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="UppQ1524o"/> on fols 1r, 2r.
+                    Division into verses is indicated with ፤ .
+                    The end of the poem is indicated with ። .
+                    The first line was originally laid out as four lines in the manuscript, each ending with ፤ .
+                However, later this was noted and the lines were marked as constituting a single line by a speaker of
+                Swedish (who noted down <foreign xml:lang="sv">"1 rad"</foreign>, "one line").
+                Judging from the rhyme, lines 4 and 5 also appear to have been divided—each into two lines—but this has not
+                been noted in the manuscript.</note>
+                <ab>
+                    <l n="1">ዓለም፡ እስከ፡ ጊዜሃ<choice><sic>፤</sic><corr resp="JK">፡</corr></choice>
+                        በሐብለ፡ ዝንጋኤ፡ ትስሕበኒ<choice><sic>፤</sic><corr resp="JK">፡</corr></choice>
+                        ወታወርደኒ፡ ካእበ<choice><sic>፤</sic><corr resp="JK">፡</corr></choice> 
+                                <del rend="overUnderlined">ሐ</del><add place="above">ኀ</add>በ፡ ዘቀዳሚ፡ መርገም፤</l>
+                    <l n="2">ለዘተወለድነ፡ በሥጋ፡ ወደም፤</l><pb n="2r"/>
```

Comment:
One line in the manuscript is missing here: ዘከእመራሰ፡ ጠቢብ፡ ላባዊ፤

Reply:
You are right! Thanks!